src/vue/Components/Theming/PreviewTheme.vue

```diff
@@ -63,6 +63,10 @@
 
 <style lang="scss">
     .theme-preview {
+        pointer-events: none;
+        border-style: inset;
+        filter: blur(1px);
```

Comment:
I don't understand the reason for this at all.
Now the preview is blurred, has an unstyled border and you can't click trough it anymore.